tests/numerics/storage.output

```diff
@@ -0,0 +1,62 @@
+DEAL::---- FIFO: 1
+DEAL::size:     0
+DEAL::max_size: 3
+DEAL::initial:
+DEAL::0.200000
+DEAL::0.100000
+DEAL::append:
+DEAL::0.555000
+DEAL::0.200000
+DEAL::0.100000
+DEAL::remove second element:
+DEAL::0.555000
+DEAL::0.100000
+DEAL::change 0th:
+DEAL::0.330000
+DEAL::0.100000
+DEAL::append:
+DEAL::0.666000
+DEAL::0.330000
+DEAL::0.100000
+DEAL::change 0th:
+DEAL::0.220000
+DEAL::0.330000
+DEAL::0.100000
+DEAL::append:
+DEAL::0.777000
+DEAL::0.220000
+DEAL::0.330000
+DEAL::remove last:
+DEAL::0.777000
+DEAL::0.220000
+DEAL::---- FIFO: 0
+DEAL::size:     0
+DEAL::max_size: 3
+DEAL::initial:
+DEAL::0.100000
+DEAL::0.200000
+DEAL::append:
+DEAL::0.100000
+DEAL::0.200000
+DEAL::0.555000
+DEAL::remove second element:
+DEAL::0.100000
+DEAL::0.555000
+DEAL::change 0th:
+DEAL::0.330000
```

Comment:
Not sure if this part of implementation should be changes, as access is agnostic to FIFO/FILO